### PR TITLE
small fixes based on previous PR review

### PR DESCRIPTION
**Pull request by sunildkumar**
This fixes a few comments Leo had on the previous PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,21 +45,30 @@ This will return the IP address of the Raspberry Pi. If this does not work, you
 
 2. Enable the camera by doing the following on your Raspberry Pi:
 	a. `sudo raspi-config` opens a menu for configuring your Raspberry Pi.
```
Comment (senorkumar) [inline README.md:47]:
```suggestion
2. Enable the camera by doing the following on your Raspberry Pi:

	a. `sudo raspi-config` opens a menu for configuring your Raspberry Pi. 
```

File: README.md [hunk 2]
```diff
@@ -47,21 +45,30 @@ This will return the IP address of the Raspberry Pi. If this does not work, you
...
+	
 	e. Reboot your Raspberry Pi
+	
 
 3. Test the camera by running `raspistill -o test.jpg` This will take a picture and save it to the file `test.jpg`. You can `scp` the file to your computer to view it: `scp <username>@<Raspberry Pi IP address>:~/test.jpg`.
 
 ## Step 3:  Run Demo
 1. Clone this repository to your Raspberry Pi.
 2. Install python libraries needed for running the code.
 	a. Run `sudo apt-get install python3-picamera` to install the library necessary to interface with the camera.
```
Comment (senorkumar) [inline README.md:63]:
```suggestion

	a. Run `sudo apt-get install python3-picamera` to install the library necessary to interface with the camera.
```

File: README.md [hunk 3]
```diff
@@ -47,21 +45,30 @@ This will return the IP address of the Raspberry Pi. If this does not work, you
...
+
 	b. Run `pip install -r requirements.txt` to install everything else.
+	
 
 3. Set environment variables
 	a. Set your Groundlight api token (replace `api_example_token` with your actual token):  `export GROUNDLIGHT_API_TOKEN=api_example_token` . You can generate a token by visiting [https://app.groundlight.ai/reef/my-account/api-tokens](https://app.groundlight.ai/reef/my-account/api-tokens).
```
Comment (senorkumar) [inline README.md:69]:
```suggestion

	a. Set your Groundlight api token (replace `api_example_token` with your actual token):  `export GROUNDLIGHT_API_TOKEN=api_example_token` . You can generate a token by visiting [https://app.groundlight.ai/reef/my-account/api-tokens](https://app.groundlight.ai/reef/my-account/api-tokens).   
```

**Review by robotrapta — APPROVED** [review]
Thanks for the updates!

**Pull request merged**